### DISCUSSION TITLE
Asterisk

### DISCUSSION THREAD

**Discussion by GarethHumbel** [YetiForceCompany/YetiForceCRM#16156] [Feature improvement]
I will be attending to connecting asterisk to latest Yetiforce. Any tips, advice? Does anyone here crack on it?
Appreciate clue bats

**Comment by mariuszkrzaczkowski** [comment]
1. see previous topics  https://github.com/YetiForceCompany/YetiForceCRM/search?q=Asterisk&type=issues
2. PBX activation panel https://gitdeveloper.yetiforce.com/index.php?module=PBX&parent=Settings&view=List
3. PBX integrations are in the catalog: https://github.com/YetiForceCompany/YetiForceCRM/tree/developer/app/Integrations/Pbx

when you turn on the PBX and click on the phone number (the user must have completed the "CRM phone extension"), the performCall method is called from the active pbx driver
https://github.com/YetiForceCompany/YetiForceCRM/blob/cc168050c20ee7082cb322f522c1a1ef2997aeeb/app/Integrations/Pbx/Base.php#L31-L36
